actions/createLUTproposal.ts

```diff
@@ -246,12 +257,15 @@ export const createLUTProposal = async (
     wallet,
     transactionInstructions: [
       {
-        instructionsSet: [
-          { transactionInstruction: lookupTableInst },
-          { transactionInstruction: extendInstruction },
-        ],
+        instructionsSet: [{ transactionInstruction: lookupTableInst }],
         sequenceType: SequenceType.Sequential,
       },
+      ...extendInstructions.map((x) => {
```

Comment:
should this be 
```
...extendInstructions.map((x, idx) => {
        return {
          instructionsSet: [{ transactionInstruction: x }],
          sequenceType: idx == 0? SequenceType.Sequential : SequenceType.Parallel,
        }
      }),
```
to prevent parallel tx won't wait till sequential tx is completed?
However, when I'm testing sendNFT, the PR code works.

Reply:
technically this could be in parallel but, because of reasons, i'd rather it be sequential.